[34m2024-08-23 18:22:41,351 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-23 18:22:41,353 - discord.client - INFO - logging in using static token[0m
[32m2024-08-23 18:22:42,247 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 3c2b73f0c9064705e499fc849eadd878).[0m
[32m2024-08-23 18:22:44,262 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-23 18:22:44,573 - cogs.close_ticket - INFO - CloseTicketCog loaded successfully.[0m
[32m2024-08-23 18:22:44,575 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-23 18:22:44,591 - cogs.create_ticket - INFO - CreateTicketCog loaded successfully.[0m
[32m2024-08-23 18:22:44,591 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-23 18:22:44,599 - cogs.ping - INFO - PingCog loaded successfully.[0m
[32m2024-08-23 18:22:44,599 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-23 18:22:44,606 - cogs.view_tickets - INFO - ViewTicketsCog loaded successfully.[0m
[32m2024-08-23 18:22:44,607 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-23 18:42:44,069 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-23 18:42:44,071 - discord.client - INFO - logging in using static token[0m
[32m2024-08-23 18:42:45,035 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: e6c180661744eb9ec1a75d229941d395).[0m
[32m2024-08-23 18:42:47,046 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-23 18:42:47,399 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-23 18:42:47,410 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-23 18:42:47,423 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-23 18:42:47,434 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 18:51:02,990 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 18:51:02,993 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 18:51:03,925 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 8dc1db653204a45f2d082c044dbc017f).[0m
[32m2024-08-24 18:51:05,944 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 18:51:08,270 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 18:51:08,287 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 18:51:08,355 - __main__ - INFO - Loaded cog: ping.py[0m
[31m2024-08-24 18:51:08,420 - __main__ - ERROR - Failed to load cog reminder.py: Extension 'cogs.reminder' raised an error: TypeError: module() takes at most 2 arguments (3 given)[0m
[32m2024-08-24 18:51:08,441 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 18:53:48,205 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 18:53:48,206 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 18:53:49,017 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 7cd904b7672d6047906f0f14a32e4855).[0m
[32m2024-08-24 18:53:51,024 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 18:53:51,329 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 18:53:51,337 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 18:53:51,344 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 18:53:51,362 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 18:53:51,369 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 18:57:10,525 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 18:57:10,527 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 18:57:11,268 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 3f8476558377613fae5114e01a0d3c7f).[0m
[32m2024-08-24 18:57:13,280 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 18:57:13,573 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 18:57:13,578 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 18:57:13,580 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 18:57:13,589 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 18:57:13,591 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[31m2024-08-24 18:57:25,495 - __main__ - ERROR - Error syncing commands: Failed to upload commands to Discord (HTTP status 400, error code 50035)
In command 'reminder' defined in function 'ReminderCog.reminder'
  description: Must be 100 or fewer in length.[0m
[34m2024-08-24 18:58:22,791 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 18:58:22,794 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 18:58:23,501 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 759eaeda622b4629186aaed0314757b5).[0m
[32m2024-08-24 18:58:25,521 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 18:58:25,818 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 18:58:25,824 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 18:58:25,827 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 18:58:25,843 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 18:58:25,846 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[32m2024-08-24 18:58:30,557 - __main__ - INFO - Commands synced globally.[0m
[34m2024-08-24 19:03:56,441 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:03:56,443 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:03:57,135 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: aa2348a96567f5d63bf3630d82281b58).[0m
[32m2024-08-24 19:03:59,154 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:03:59,448 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:03:59,454 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:03:59,456 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:03:59,474 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:03:59,476 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:06:06,301 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:06:06,303 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:06:07,028 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 776bc4c937aea6a7f0c01a80e921a2e4).[0m
[32m2024-08-24 19:06:09,043 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:06:09,341 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:06:09,346 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:06:09,349 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:06:09,366 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:06:09,368 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:06:33,041 - cogs.reminder - DEBUG - Attempting to defer interaction...[0m
[34m2024-08-24 19:06:33,388 - cogs.reminder - DEBUG - Interaction deferred successfully.[0m
[34m2024-08-24 19:06:38,134 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[34m2024-08-24 19:08:08,936 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:08:08,938 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:08:09,675 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 4dfffd52d5ef46bf2800568a181ca807).[0m
[32m2024-08-24 19:08:11,692 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:08:12,032 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:08:12,037 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:08:12,039 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:08:12,058 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:08:12,060 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:08:28,854 - cogs.reminder - DEBUG - Attempting to defer interaction...[0m
[34m2024-08-24 19:08:29,135 - cogs.reminder - DEBUG - Interaction deferred successfully.[0m
[34m2024-08-24 19:08:33,917 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[34m2024-08-24 19:10:45,456 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:10:45,458 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:10:46,145 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 29d614080986b88c197380bbfd24a147).[0m
[32m2024-08-24 19:10:48,153 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:10:48,464 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:10:48,469 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:10:48,472 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:10:48,490 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:10:48,492 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:11:08,209 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[33m2024-08-24 19:15:08,217 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 19:24:46,921 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:24:46,923 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:24:47,876 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 0d92557d1a0eb6890d4978e540ba8010).[0m
[32m2024-08-24 19:24:49,905 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:24:50,209 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:24:50,214 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:24:50,217 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:24:50,235 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:24:50,237 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:26:55,212 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:26:55,215 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:26:55,868 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 64db9081181288faee7a32c23b897f80).[0m
[32m2024-08-24 19:26:57,908 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:26:58,214 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:26:58,220 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:26:58,223 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:26:58,235 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:26:58,237 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:27:32,235 - cogs.reminder - DEBUG - Reminder set for message ID 1277061769130414160[0m
[31m2024-08-24 19:27:32,236 - cogs.reminder - ERROR - Error updating reminder embed: 'NoneType' object has no attribute 'id'[0m
Traceback (most recent call last):
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 84, in update_embed
    reminder_message_id = interaction.message.id
AttributeError: 'NoneType' object has no attribute 'id'
[34m2024-08-24 19:27:40,088 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[34m2024-08-24 19:27:40,187 - cogs.reminder - DEBUG - Reminder found and canceled for message ID 1277061769130414160[0m
[34m2024-08-24 19:30:32,262 - cogs.reminder - DEBUG - Reminder was canceled for message ID 1277061769130414160[0m
[34m2024-08-24 19:34:49,908 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:34:49,909 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:34:50,544 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 2061d9877fe5390e7e5768b79890c1c4).[0m
[32m2024-08-24 19:34:52,577 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:34:52,885 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:34:52,891 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:34:52,893 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:34:52,913 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:34:52,915 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:36:41,334 - cogs.reminder - DEBUG - Reminder set for message ID 1277064072533377024[0m
[34m2024-08-24 19:36:53,452 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[34m2024-08-24 19:36:53,641 - cogs.reminder - DEBUG - Reminder found and canceled for message ID 1277064072533377024[0m
[34m2024-08-24 19:37:21,343 - cogs.reminder - DEBUG - Reminder was canceled for message ID 1277064072533377024[0m
[34m2024-08-24 19:37:31,802 - cogs.reminder - DEBUG - Reminder set for message ID 1277064284156727306[0m
[34m2024-08-24 19:38:16,799 - cogs.reminder - DEBUG - Sending reminder for message ID 1277064284156727306[0m
[33m2024-08-24 19:39:16,807 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 19:40:30,510 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:40:30,512 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:40:31,221 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: aed41db57650b74454df3fe322c0d538).[0m
[32m2024-08-24 19:40:33,236 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:40:33,555 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:40:33,559 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:40:33,562 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:40:33,582 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:40:33,584 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:41:47,587 - cogs.reminder - DEBUG - Reminder set for message ID 1277065356594118700[0m
[34m2024-08-24 19:42:17,592 - cogs.reminder - DEBUG - Sending reminder for message ID 1277065356594118700[0m
[34m2024-08-24 19:42:25,553 - cogs.reminder - DEBUG - Cancel button clicked by 274005218516271104[0m
[34m2024-08-24 19:42:25,681 - cogs.reminder - DEBUG - No active reminder found for message ID 1277065356594118700[0m
[33m2024-08-24 19:43:55,557 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 19:45:43,532 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:45:43,534 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:45:44,252 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 19e524a9f79d0b40e119385bc0d03b06).[0m
[32m2024-08-24 19:45:46,272 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:45:46,587 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:45:46,592 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:45:46,595 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:45:46,616 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:45:46,618 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:48:46,055 - cogs.reminder - DEBUG - Reminder set for message ID 1277067111893569620[0m
[34m2024-08-24 19:49:46,066 - cogs.reminder - DEBUG - Sending reminder for message ID 1277067111893569620[0m
[33m2024-08-24 19:50:46,064 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 19:52:07,464 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 19:52:07,466 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 19:52:08,162 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: f3dd0e0205aa03a34947684623c04f32).[0m
[32m2024-08-24 19:52:10,169 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 19:52:10,505 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 19:52:10,509 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 19:52:10,512 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 19:52:10,532 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 19:52:10,535 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 19:52:40,895 - cogs.reminder - DEBUG - Reminder set for message ID 1277068096330272769[0m
[34m2024-08-24 19:53:40,915 - cogs.reminder - DEBUG - Sending reminder for message ID 1277068096330272769[0m
[33m2024-08-24 19:54:40,898 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 20:00:49,586 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:00:49,587 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:00:50,397 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: e8eed5404e18ae6dfbe1d53c99a85d0d).[0m
[32m2024-08-24 20:00:52,402 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:00:52,718 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:00:52,723 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:00:52,727 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 20:00:52,747 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 20:00:52,749 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[33m2024-08-24 20:03:17,581 - cogs.reminder - WARNING - Time conversion error: Time limit exceeded. Please set a reminder for 24 hours or less.[0m
[31m2024-08-24 20:03:20,607 - discord.app_commands.tree - ERROR - Ignoring exception in command 'reminder'[0m
Traceback (most recent call last):
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 61, in reminder
    reminder_time = self.convert_time(time)
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 34, in convert_time
    raise ValueError("Time limit exceeded. Please set a reminder for 24 hours or less.")
ValueError: Time limit exceeded. Please set a reminder for 24 hours or less.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\app_commands\commands.py", line 841, in _do_call
    return await self._callback(self.binding, interaction, **params)  # type: ignore
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 106, in reminder
    await interaction.followup.send(str(e), ephemeral=True)
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\webhook\async_.py", line 1800, in send
    data = await adapter.execute_webhook(
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\webhook\async_.py", line 219, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10015): Unknown Webhook

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\app_commands\tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\app_commands\commands.py", line 867, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
  File "C:\Users\Norah\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\discord\app_commands\commands.py", line 860, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'reminder' raised an exception: NotFound: 404 Not Found (error code: 10015): Unknown Webhook
[34m2024-08-24 20:07:07,823 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:07:07,825 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:07:08,533 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 0094bae80ade6d0b2139bdb1fda4ee7f).[0m
[32m2024-08-24 20:07:10,557 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:07:10,858 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:07:10,864 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:07:10,867 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 20:07:10,886 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 20:07:10,888 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[33m2024-08-24 20:07:17,171 - cogs.reminder - WARNING - Time conversion error: Time limit exceeded. Please set a reminder for 24 hours or less.[0m
[33m2024-08-24 20:09:08,542 - cogs.reminder - WARNING - Cancel button interaction timed out.[0m
[34m2024-08-24 20:24:46,344 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:24:46,346 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:24:46,963 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: bc295909a41373be87725d7dd857541f).[0m
[32m2024-08-24 20:24:48,968 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:24:49,269 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:24:49,275 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:24:49,278 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 20:24:49,298 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 20:24:49,300 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[31m2024-08-24 20:25:24,311 - cogs.reminder - ERROR - Error updating reminder embed: 'embed_message'[0m
Traceback (most recent call last):
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 151, in update_embed
    embed = reminder["embed_message"].embeds[0]
KeyError: 'embed_message'
[34m2024-08-24 20:28:04,090 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:28:04,092 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:28:04,785 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 2e156c2df6b43583565e69c52f1aeb00).[0m
[32m2024-08-24 20:28:06,822 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:28:07,123 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:28:07,129 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:28:07,132 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 20:28:07,144 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 20:28:07,146 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 20:55:05,867 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:55:05,869 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:55:06,871 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 595f59fa87b8863fe2d15cc51cb1f0f0).[0m
[32m2024-08-24 20:55:08,916 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:55:09,341 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:55:09,345 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:55:09,349 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 20:55:09,423 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 20:55:09,425 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[31m2024-08-24 20:55:09,425 - cogs.reminder - ERROR - Failed to resume reminder: 'channel_id'[0m
Traceback (most recent call last):
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 41, in resume_reminder
    channel = self.bot.get_channel(reminder["channel_id"])
KeyError: 'channel_id'
[31m2024-08-24 20:55:09,427 - cogs.reminder - ERROR - Error updating reminder embed: 'channel_id'[0m
Traceback (most recent call last):
  File "C:\Users\Norah\Documents\GitHub\NorahBot\cogs\reminder.py", line 174, in update_embed
    channel = self.bot.get_channel(reminder["channel_id"])
KeyError: 'channel_id'
[34m2024-08-24 20:56:19,256 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:56:19,258 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:56:19,890 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 5ebf239c898a8061c7d01cc42fec24d6).[0m
[32m2024-08-24 20:56:21,900 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:56:22,219 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:56:22,225 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:56:22,227 - __main__ - INFO - Loaded cog: ping.py[0m
[31m2024-08-24 20:56:22,239 - __main__ - ERROR - Failed to load cog reminder.py: Extension 'cogs.reminder' raised an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)[0m
[32m2024-08-24 20:56:22,242 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 20:57:39,992 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:57:39,995 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:57:40,766 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 9d19ed56578edd173ec845bf3f80e4bc).[0m
[32m2024-08-24 20:57:42,790 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 20:57:43,123 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 20:57:43,129 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 20:57:43,133 - __main__ - INFO - Loaded cog: ping.py[0m
[31m2024-08-24 20:57:43,151 - __main__ - ERROR - Failed to load cog reminder.py: Extension 'cogs.reminder' raised an error: JSONDecodeError: Expecting value: line 1 column 1 (char 0)[0m
[32m2024-08-24 20:57:43,153 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 20:59:57,455 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 20:59:57,457 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 20:59:58,164 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: c702f56488c350ef2e5bba0509483d8d).[0m
[32m2024-08-24 21:00:00,177 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 21:00:00,530 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 21:00:00,536 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 21:00:00,538 - __main__ - INFO - Loaded cog: ping.py[0m
[33m2024-08-24 21:00:00,560 - cogs.reminder - WARNING - reminders.json is empty or corrupted. Reinitializing as an empty dictionary.[0m
[32m2024-08-24 21:00:00,561 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 21:00:00,563 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 21:00:40,913 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 21:00:40,915 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 21:00:41,558 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 78b700cdd6525eb6283d23ef945b5f2d).[0m
[32m2024-08-24 21:00:43,573 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 21:00:43,895 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 21:00:43,901 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 21:00:43,903 - __main__ - INFO - Loaded cog: ping.py[0m
[33m2024-08-24 21:00:43,926 - cogs.reminder - WARNING - reminders.json is empty or corrupted. Reinitializing as an empty dictionary.[0m
[32m2024-08-24 21:00:43,927 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 21:00:43,929 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-08-24 21:02:04,342 - asyncio - DEBUG - Using proactor: IocpProactor[0m
[32m2024-08-24 21:02:04,344 - discord.client - INFO - logging in using static token[0m
[32m2024-08-24 21:02:05,048 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 7ef02fc49590fe6ae5df351bee0560c9).[0m
[32m2024-08-24 21:02:07,074 - __main__ - INFO - Logged in as Vigil (1260362601032978525)[0m
[32m2024-08-24 21:02:07,421 - __main__ - INFO - Loaded cog: close_ticket.py[0m
[32m2024-08-24 21:02:07,426 - __main__ - INFO - Loaded cog: create_ticket.py[0m
[32m2024-08-24 21:02:07,428 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-08-24 21:02:07,438 - __main__ - INFO - Loaded cog: reminder.py[0m
[32m2024-08-24 21:02:07,440 - __main__ - INFO - Loaded cog: view_tickets.py[0m
[34m2024-11-04 20:15:43,118 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:15:43,121 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:15:46,944 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 1555e84743c756f3c3a44e7197575943).[0m
[32m2024-11-04 20:15:48,950 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 20:15:48,955 - __main__ - INFO - Loaded cog: ping.py[0m
[34m2024-11-04 20:17:14,807 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:17:14,808 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:17:17,695 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 3777df1be1ded63d7ee821f02ebd5148).[0m
[32m2024-11-04 20:17:19,698 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 20:17:19,702 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 20:17:29,640 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-04 20:18:20,496 - cogs.ping - INFO - Ping command executed. Latency: 289ms[0m
[32m2024-11-04 20:18:25,421 - cogs.ping - INFO - Ping command executed. Latency: 289ms[0m
[34m2024-11-04 20:33:55,451 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:33:55,452 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:33:59,487 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 012ae3ce5f77ba00bd832122abb5e6c9).[0m
[32m2024-11-04 20:34:01,493 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 20:34:01,501 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 20:34:21,841 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-04 20:35:28,106 - cogs.ping - INFO - Ping command executed. Latency: 330ms[0m
[34m2024-11-04 20:57:47,129 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:57:47,130 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:57:50,075 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 892a29c80bda39f89db372a46916b3f3).[0m
[32m2024-11-04 20:57:52,082 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[31m2024-11-04 20:57:52,101 - __main__ - ERROR - Failed to load cog ping.py: Extension 'cogs.ping' raised an error: ValueError: 'Say Hi' must be between 1-32 characters and contain only lower-case letters, numbers, hyphens, or underscores.[0m
[31m2024-11-04 20:57:52,104 - __main__ - ERROR - Failed to load cog sayhi.py: Extension 'cogs.sayhi' raised an error: ValueError: 'SayHi' must be all lower-case[0m
[34m2024-11-04 20:58:30,381 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:58:30,381 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:58:35,401 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: afac14c8fd293072b96996b07bf965f9).[0m
[32m2024-11-04 20:58:37,414 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 20:58:37,423 - __main__ - INFO - Loaded cog: ping.py[0m
[31m2024-11-04 20:58:37,432 - __main__ - ERROR - Failed to load cog sayhi.py: Extension 'cogs.sayhi' raised an error: ValueError: 'SayHi' must be all lower-case[0m
[34m2024-11-04 20:59:01,797 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 20:59:01,798 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 20:59:04,950 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: f77e20242602526630e5131085e782b6).[0m
[32m2024-11-04 20:59:06,958 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 20:59:06,960 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 20:59:06,964 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-04 20:59:23,783 - __main__ - INFO - Commands synced globally.[0m
[34m2024-11-04 21:17:45,834 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 21:17:45,835 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 21:17:49,066 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 9b99f2402ee42854adfe830729104cf7).[0m
[32m2024-11-04 21:17:51,075 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 21:17:51,083 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 21:17:51,089 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-04 21:18:01,228 - cogs.ping - INFO - Ping command executed. Latency: 299ms[0m
[34m2024-11-04 22:32:52,496 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 22:32:52,497 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 22:32:55,634 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: d7d2483100d8b5841bd33f6b446e8a76).[0m
[32m2024-11-04 22:32:57,646 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 22:32:57,653 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 22:32:57,658 - __main__ - INFO - Loaded cog: sayhi.py[0m
[31m2024-11-04 22:32:57,669 - __main__ - ERROR - Failed to load cog timestamp.py: Extension 'cogs.timestamp' has no 'setup' function.[0m
[32m2024-11-04 22:33:10,757 - __main__ - INFO - Commands synced globally.[0m
[31m2024-11-04 22:33:24,334 - __main__ - ERROR - An error occurred: Command "timestamp" is not found[0m
[32m2024-11-04 22:36:35,691 - __main__ - INFO - Commands synced globally.[0m
[34m2024-11-04 22:41:05,170 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 22:41:05,171 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 22:41:12,448 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 40a2f2ae8560dd035fd768b8b836b27e).[0m
[32m2024-11-04 22:41:14,456 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 22:41:14,462 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 22:41:14,470 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-04 22:41:14,484 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-04 22:41:26,854 - __main__ - INFO - Commands synced globally.[0m
[34m2024-11-04 23:01:55,895 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 23:01:55,902 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 23:01:59,751 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 62645d4ab7eb6f95302134eaaa0d4d64).[0m
[32m2024-11-04 23:02:01,759 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 23:02:01,767 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 23:02:01,770 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-04 23:02:01,797 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-04 23:02:18,720 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-04 23:03:50,968 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC[0m
[32m2024-11-04 23:05:51,763 - discord.gateway - INFO - Shard ID None has successfully RESUMED session 62645d4ab7eb6f95302134eaaa0d4d64.[0m
[32m2024-11-04 23:06:12,583 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC[0m
[34m2024-11-04 23:19:33,618 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-04 23:19:33,618 - discord.client - INFO - logging in using static token[0m
[32m2024-11-04 23:19:36,538 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 4fc6eae3ec8f9492a4e780aaebaa08dd).[0m
[32m2024-11-04 23:19:38,542 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-04 23:19:38,546 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-04 23:19:38,548 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-04 23:19:38,560 - __main__ - INFO - Loaded cog: timestamp.py[0m
[34m2024-11-05 14:26:15,447 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 14:26:15,448 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 14:26:17,877 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 441b677b66aea670944712eb18fd70b0).[0m
[32m2024-11-05 14:26:19,885 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 14:26:19,890 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 14:26:19,894 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 14:26:19,911 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 14:26:29,720 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-05 14:26:51,894 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC[0m
[31m2024-11-05 14:27:35,641 - cogs.timestamp - ERROR - Failed to execute timestamp command: 'builtin_function_or_method' object has no attribute 'timestamp'[0m
[32m2024-11-05 14:28:05,855 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC[0m
[31m2024-11-05 14:30:37,021 - discord.client - ERROR - Attempting a reconnect in 0.30s[0m
Traceback (most recent call last):
  File "/home/pyguy/Documents/time-stamp-bot/.venv/lib/python3.10/site-packages/discord/client.py", line 659, in connect
    await self.ws.poll_event()
  File "/home/pyguy/Documents/time-stamp-bot/.venv/lib/python3.10/site-packages/discord/gateway.py", line 646, in poll_event
    raise ConnectionClosed(self.socket, shard_id=self.shard_id, code=code) from None
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
[32m2024-11-05 14:30:41,795 - discord.gateway - INFO - Shard ID None has successfully RESUMED session 441b677b66aea670944712eb18fd70b0.[0m
[32m2024-11-05 14:39:44,306 - discord.gateway - INFO - Shard ID None has successfully RESUMED session 441b677b66aea670944712eb18fd70b0.[0m
[33m2024-11-05 14:47:36,279 - discord.gateway - WARNING - Can't keep up, shard ID None websocket is 21.4s behind.[0m
[34m2024-11-05 22:08:02,433 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:08:02,433 - discord.client - INFO - logging in using static token[0m
[34m2024-11-05 22:10:13,638 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:10:13,638 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 22:10:17,670 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 10b42a08c93969ebd35e00db1d06ad82).[0m
[32m2024-11-05 22:10:19,678 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 22:10:19,684 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 22:10:19,688 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 22:10:19,713 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 22:10:39,778 - __main__ - INFO - Commands synced globally.[0m
[31m2024-11-05 22:11:29,773 - cogs.timestamp - ERROR - Failed to execute timestamp command: 'builtin_function_or_method' object has no attribute 'timestamp'[0m
[31m2024-11-05 22:14:08,709 - cogs.timestamp - ERROR - Failed to execute timestamp command: 'builtin_function_or_method' object has no attribute 'timestamp'[0m
[34m2024-11-05 22:14:32,140 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:14:32,141 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 22:14:35,314 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 00d013c860b6e3abd5d05500ce677023).[0m
[32m2024-11-05 22:14:37,324 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 22:14:37,329 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 22:14:37,332 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 22:14:37,349 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 22:14:46,910 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-05 22:19:28,648 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, format: short[0m
[32m2024-11-05 22:19:54,490 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, format: relative[0m
[34m2024-11-05 22:22:41,396 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:22:41,397 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 22:22:44,878 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 9b6006344fca5d1766ab5a2ea1b0ca41).[0m
[32m2024-11-05 22:22:46,888 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 22:22:46,896 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 22:22:46,901 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 22:22:46,941 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 22:23:25,197 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:30, format: relative[0m
[34m2024-11-05 22:47:49,036 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:47:49,037 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 22:47:51,860 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 1e62c74aaffd5b71120730db3c861903).[0m
[32m2024-11-05 22:47:53,869 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 22:47:53,873 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 22:47:53,876 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 22:47:53,894 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 22:48:46,590 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:50, format: relative[0m
[32m2024-11-05 22:49:11,179 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 21:50, format: relative[0m
[32m2024-11-05 22:50:25,466 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 11:50, format: relative[0m
[32m2024-11-05 22:52:12,172 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 12:50, format: full[0m
[34m2024-11-05 22:55:45,467 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 22:55:45,467 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 22:55:50,438 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: adca8278c2253ac59669587a226017ad).[0m
[32m2024-11-05 22:55:52,445 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 22:55:52,450 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 22:55:52,452 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 22:55:52,470 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 22:55:54,854 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-05 22:57:08,265 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:58, format: full[0m
[32m2024-11-05 23:02:29,391 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 23:02, format: full[0m
[34m2024-11-05 23:05:49,226 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 23:05:49,227 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 23:05:51,785 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 9aed060d564f83caa2aa5b70152ed7eb).[0m
[32m2024-11-05 23:05:53,795 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 23:05:53,801 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 23:05:53,805 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 23:05:53,824 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 23:05:58,198 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-05 23:06:54,373 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:10, format: full[0m
[32m2024-11-05 23:07:30,234 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, format: full[0m
[32m2024-11-05 23:07:51,074 - cogs.timestamp - INFO - Timestamp command executed for timezone: GMT at time: current time, format: full[0m
[32m2024-11-05 23:08:15,832 - cogs.timestamp - INFO - Timestamp command executed for timezone: GMT at time: 23:04, format: full[0m
[34m2024-11-05 23:09:05,750 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-05 23:09:05,750 - discord.client - INFO - logging in using static token[0m
[32m2024-11-05 23:09:09,414 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: d009ad2267050dcd50895e96be75fa39).[0m
[32m2024-11-05 23:09:11,422 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-05 23:09:11,425 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-05 23:09:11,428 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-05 23:09:11,446 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-05 23:09:19,710 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-05 23:09:39,061 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:00, format: full[0m
[32m2024-11-05 23:12:14,781 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 23:14, format: relative[0m
[34m2024-11-07 19:48:00,019 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 19:48:00,020 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 19:48:08,378 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: db7ad80243fc167978ac856676b93a39).[0m
[32m2024-11-07 19:48:10,388 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 19:48:10,393 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 19:48:10,397 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 19:48:10,413 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 19:49:55,557 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 19:50, format: relative[0m
[34m2024-11-07 19:53:55,234 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 19:53:55,235 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 19:53:58,001 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: b2edd5eb905ac0b0103100d8c149a9fd).[0m
[32m2024-11-07 19:54:00,009 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 19:54:00,012 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 19:54:00,013 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 19:54:00,021 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 19:54:27,853 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:30, format: relative[0m
[32m2024-11-07 19:54:37,680 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 22:30, format: full[0m
[32m2024-11-07 20:08:26,565 - discord.gateway - INFO - Shard ID None has successfully RESUMED session b2edd5eb905ac0b0103100d8c149a9fd.[0m
[34m2024-11-07 20:22:28,611 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 20:22:28,611 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 20:22:32,709 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 4c5172b4896e0be98285d22de5e13d7e).[0m
[32m2024-11-07 20:22:34,750 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 20:22:34,759 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 20:22:34,768 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 20:22:34,813 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 20:22:49,208 - __main__ - INFO - Commands synced globally.[0m
[31m2024-11-07 20:25:05,085 - cogs.timestamp - ERROR - Failed to execute timestamp command: local variable 'current_date' referenced before assignment[0m
[32m2024-11-07 20:28:54,867 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, format: full[0m
[31m2024-11-07 20:29:43,400 - cogs.timestamp - ERROR - Failed to execute timestamp command: local variable 'current_date' referenced before assignment[0m
[34m2024-11-07 20:34:13,245 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 20:34:13,246 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 20:34:15,583 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 7d7c2b037f834397e197a77a0b90c35c).[0m
[32m2024-11-07 20:34:17,594 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 20:34:17,599 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 20:34:17,603 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 20:34:17,639 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 20:34:25,969 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 20:35:44,204 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-01, format: full[0m
[32m2024-11-07 20:36:11,752 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-11-01, format: full[0m
[34m2024-11-07 20:37:53,093 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 20:37:53,094 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 20:37:55,671 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: f5e0b36eb189e9c9170d61ae627f6f31).[0m
[32m2024-11-07 20:37:57,678 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 20:37:57,682 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 20:37:57,685 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 20:37:57,706 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 20:38:17,876 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 20:39:14,470 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-01, format: full[0m
[32m2024-11-07 20:39:36,198 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-02, format: full[0m
[32m2024-11-07 20:40:09,770 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-00, format: full[0m
[32m2024-11-07 20:40:24,236 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-05, format: full[0m
[34m2024-11-07 21:02:27,692 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:02:27,693 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:02:29,942 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: bb0428bdae8740fa19ecec573a199e6a).[0m
[32m2024-11-07 21:02:31,962 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:02:31,967 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:02:31,970 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:02:31,988 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:03:16,855 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:04:31,862 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-02, format: full[0m
[34m2024-11-07 21:05:54,555 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:05:54,556 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:05:58,897 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 734ae5db4105894ffaecc1d5c280f98b).[0m
[32m2024-11-07 21:06:00,910 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:06:00,915 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:06:00,918 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:06:00,935 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:06:05,029 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:06:41,336 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: current date, format: full[0m
[32m2024-11-07 21:06:56,385 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-01, format: full[0m
[32m2024-11-07 21:07:14,060 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:07:24,655 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-2, format: full[0m
[32m2024-11-07 21:07:32,606 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-15, format: full[0m
[32m2024-11-07 21:08:16,858 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[32m2024-11-07 21:08:36,115 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[34m2024-11-07 21:09:41,055 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:09:41,056 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:09:43,646 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: c391d42f068c82239568f0973aacaf09).[0m
[32m2024-11-07 21:09:45,678 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:09:45,683 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:09:45,687 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:09:45,709 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:09:50,109 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:12:16,631 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-2, format: full[0m
[32m2024-11-07 21:12:38,474 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[34m2024-11-07 21:15:49,637 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:15:49,637 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:15:55,152 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 9319564ad2a3baf241d11a429b6384f9).[0m
[32m2024-11-07 21:15:57,160 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:15:57,164 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:15:57,167 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:15:57,177 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:16:03,795 - __main__ - INFO - Commands synced globally.[0m
[31m2024-11-07 21:16:52,534 - discord.app_commands.tree - ERROR - Ignoring exception in command 'timestamp'[0m
Traceback (most recent call last):
  File "/home/pyguy/Documents/time-stamp-bot/cogs/timestamp.py", line 43, in timestamp
    custom_date = datetime.date(year, month, day+31)  # For some reason this lets the program function please don't ask
ValueError: day is out of range for month

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pyguy/Documents/time-stamp-bot/cogs/timestamp.py", line 48, in timestamp
    await interaction.response.send_message("Invalid date format. Please use `YYYY-MM-DD` format.")
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/interactions.py", line 778, in send_message
    await adapter.create_interaction_response(
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/webhook/async_.py", line 219, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/app_commands/commands.py", line 841, in _do_call
    return await self._callback(self.binding, interaction, **params)  # type: ignore
  File "/home/pyguy/Documents/time-stamp-bot/cogs/timestamp.py", line 96, in timestamp
    await interaction.response.send_message("An error occurred while processing the timestamp command.")
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/interactions.py", line 778, in send_message
    await adapter.create_interaction_response(
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/webhook/async_.py", line 219, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/app_commands/tree.py", line 1248, in _call
    await command._invoke_with_namespace(interaction, namespace)
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/app_commands/commands.py", line 867, in _invoke_with_namespace
    return await self._do_call(interaction, transformed_values)
  File "/home/pyguy/Documents/NorahBot-main/.venv/lib/python3.10/site-packages/discord/app_commands/commands.py", line 860, in _do_call
    raise CommandInvokeError(self, e) from e
discord.app_commands.errors.CommandInvokeError: Command 'timestamp' raised an exception: NotFound: 404 Not Found (error code: 10062): Unknown interaction
[34m2024-11-07 21:17:05,016 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:17:05,017 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:17:10,420 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: ad6a92f357a2720ddaf3bb646d8a4d19).[0m
[32m2024-11-07 21:17:12,426 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:17:12,431 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:17:12,437 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:17:12,457 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:17:28,087 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:19:19,920 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[32m2024-11-07 21:19:28,338 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-31, format: full[0m
[32m2024-11-07 21:20:59,035 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:21:29,660 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-02, format: full[0m
[32m2024-11-07 21:21:34,612 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-03, format: full[0m
[32m2024-11-07 21:21:54,834 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[32m2024-11-07 21:22:24,261 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:23:25,268 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-30, format: full[0m
[32m2024-11-07 21:27:03,114 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-31, format: full[0m
[32m2024-11-07 21:27:12,294 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-32, format: full[0m
[32m2024-11-07 21:28:01,455 - __main__ - INFO - Commands synced globally.[0m
[34m2024-11-07 21:28:15,007 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:28:15,008 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:28:18,281 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 5e650c21caa836523a0123321d1762e7).[0m
[32m2024-11-07 21:28:20,286 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:28:20,289 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:28:20,292 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:28:20,300 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:28:58,534 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-01, format: full[0m
[32m2024-11-07 21:29:01,421 - __main__ - INFO - Commands synced globally.[0m
[34m2024-11-07 21:29:45,911 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:29:45,912 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:29:48,706 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 2e0a97ab974fff8cc0d3c963007a2dff).[0m
[32m2024-11-07 21:29:50,713 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:29:50,717 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:29:50,720 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:29:50,738 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:29:51,401 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:30:24,447 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-01, format: full[0m
[32m2024-11-07 21:30:34,168 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-02, format: full[0m
[32m2024-11-07 21:30:38,913 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-02, format: full[0m
[32m2024-11-07 21:30:42,928 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-03, format: full[0m
[34m2024-11-07 21:31:33,972 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:31:33,972 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:31:37,285 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: eae524e69500b7128fadad886868f6f9).[0m
[32m2024-11-07 21:31:39,290 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:31:39,291 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:31:39,292 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:31:39,298 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:31:43,171 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:31:56,489 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-01, format: full[0m
[32m2024-11-07 21:32:01,314 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-02, format: full[0m
[32m2024-11-07 21:32:05,824 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-03, format: full[0m
[34m2024-11-07 21:32:46,376 - asyncio - DEBUG - Using selector: EpollSelector[0m
[32m2024-11-07 21:32:46,377 - discord.client - INFO - logging in using static token[0m
[32m2024-11-07 21:32:49,365 - discord.gateway - INFO - Shard ID None has connected to Gateway (Session ID: 536eed841d547cceb2842dc0f706f112).[0m
[32m2024-11-07 21:32:51,369 - __main__ - INFO - Logged in as Fanta Lite (1303063696578187285)[0m
[32m2024-11-07 21:32:51,374 - __main__ - INFO - Loaded cog: ping.py[0m
[32m2024-11-07 21:32:51,376 - __main__ - INFO - Loaded cog: sayhi.py[0m
[32m2024-11-07 21:32:51,392 - __main__ - INFO - Loaded cog: timestamp.py[0m
[32m2024-11-07 21:33:42,440 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:33:59,796 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-3, format: full[0m
[32m2024-11-07 21:34:12,372 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:34:15,736 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:34:22,102 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: current time, date: 2024-12-30, format: full[0m
[32m2024-11-07 21:34:39,426 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-30, format: full[0m
[32m2024-11-07 21:34:54,789 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:35:13,465 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:01, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:35:42,994 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 23:59, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:36:00,316 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 24:00, date: 2024-12-1, format: full[0m
[32m2024-11-07 21:36:19,604 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 24:00, date: 2024-11-30, format: full[0m
[32m2024-11-07 21:36:30,313 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 24:00, date: 2024-11-30, format: relative[0m
[32m2024-11-07 21:45:56,355 - __main__ - INFO - Commands synced globally.[0m
[32m2024-11-07 21:46:31,656 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-11-30, format: relative[0m
[32m2024-11-07 21:50:42,267 - cogs.timestamp - INFO - Timestamp command executed for timezone: UTC at time: 00:00, date: 2024-11-30, format: relative[0m
